man/Makefile

```diff
@@ -90,7 +90,7 @@ cites := ${subst .,(,${addsuffix ),${man_pages}}}
 # ctags(1) tags(5) ...
 # => -e 's/\<ctags(1)/:ref:`& <&>`/g' -e 's/\<ctags(5)/:ref:`& <&>`/g' ...
 #
-is_gnu_sed := $(shell sed --version | grep -q GNU && echo 1)
+is_gnu_sed := $(shell sed --version 2> /dev/null | grep -q GNU && echo 1)
```

Comment:
Could you make a separated commit for this change?
I guess you fix another issue in this line.
The change is small so you can add a commit for this line to this pull request.

Reply:
Yes, this change is nothing to do with this PR. I will commit this change in another pull request.